src/Polly/Caching/ContextualTtl.cs

```diff
@@ -28,6 +27,11 @@ public class ContextualTtl : ITtlStrategy
     /// <returns>TimeSpan.</returns>
     public Ttl GetTtl(Context context, object? result)
     {
+        if (context is null)
```

Comment:
@martincostello
Which is better to use? "==" or the pattern "is"
The "is" mapping pattern works faster, but mostly the code uses "=="

Reply:
I would use `is` for new code where it is equivalent to `== null`. Don't worry about updating existing code to match - it was written before `is` was a thing.